__tests__/transformdata.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from '@jest/globals';
+import { transformJsonData } from '@/transform';
```

Comment:
To fix the `Module not found` error when running Jest, the `jest.config.ts` needs to be modified:

Replace the current value of `moduleNameMapper` with the following: `moduleNameMapper: { '^@/(.*)$': '<rootDir>/src/$1' },
`

Reasoning: https://stackoverflow.com/a/72404459